code/game/machinery/hologram.dm

```diff
@@ -49,6 +49,12 @@ Possible to do for anyone motivated enough:
 	var/datum/holocall/outgoing_call	//do not modify the datums only check and call the public procs
 	var/static/force_answer_call = FALSE	//Calls will be automatically answered after a couple rings, here for debugging
 	var/static/list/holopads = list()
+	var/obj/item/disk/holodisk/disk //Record disk
+	var/replay_mode = FALSE //currently replaying a recording
+	var/record_mode = FALSE //currently recording
+	var/record_start = 0  	//recording start time
+	var/record_user			//user that inititiated the recording
+	var/obj/effect/overlay/holo_pad_hologram/replay_holo	//replay hologram

```

Comment:
Handle in Destroy

Reply:
already handled by replay_stop, i'd rather not add redundancies for when the whole thing breaks.